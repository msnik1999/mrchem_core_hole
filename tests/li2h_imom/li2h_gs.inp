# vim:syntax=sh:

world_prec = 1.0e-3                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n

MPI {
  numerically_exact = true          # Guarantee identical results in MPI
}

Molecule {
  charge = 1
$coords
Li    -5.46737452    2.65708721    0.38412463
Li     2.48697407    1.97066310   -0.29143356
H     -0.09374931   -1.62164958   -3.76992803
$end
}

WaveFunction {
  method = PBE                      # Wave function method (HF or DFT)
}

SCF {
  run = false
  localize = true                   # Use canonical of localized orbitals
  write_orbitals = true
  path_orbitals = initial_guess
}