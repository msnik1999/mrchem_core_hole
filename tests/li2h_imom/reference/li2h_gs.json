{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 1,
      "coords": [
        {
          "atom": "li",
          "r_rms": 4.0992133976e-05,
          "xyz": [
            -5.46737452,
            2.65708721,
            0.38412463
          ]
        },
        {
          "atom": "li",
          "r_rms": 4.0992133976e-05,
          "xyz": [
            2.48697407,
            1.9706631,
            -0.29143356
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            -0.09374931,
            -1.62164958,
            -3.76992803
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "gs",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (PBE)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "occupancies": [],
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "write_orbitals": {
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -1.3956707147894085,
        2.047468923584971,
        -0.2119882646732089
      ],
      "charge": 1,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 7.3215341715203675,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -3.0927003058667193,
            5.7256715625183405,
            3.355108433568299
          ],
          "vector_el": [
            5.942250354133021,
            -6.535929787480799,
            6.846963253567953
          ],
          "vector_nuc": [
            -9.03495065999974,
            12.26160134999914,
            -3.491854819999654
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Li",
          "xyz": [
            -5.46737452,
            2.65708721,
            0.38412463
          ]
        },
        {
          "symbol": "Li",
          "xyz": [
            2.48697407,
            1.9706631,
            -0.29143356
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -0.09374931,
            -1.62164958,
            -3.76992803
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -2.1953822986412757,
          -2.1243921287502467,
          -0.12180598915148583
        ],
        "occupation": [
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -8.883160833086016
      },
      "orbital_positions": [
        [
          -5.4656348250806595,
          2.6569155539373233,
          0.3839517492274577
        ],
        [
          2.4853252114419644,
          1.970807515613723,
          -0.29129136431061525
        ],
        [
          0.0091844365721864,
          -1.3597581758106476,
          -3.5161420117008193
        ]
      ],
      "scf_energy": {
        "E_ee": 9.309571677795763,
        "E_eext": 0.0,
        "E_el": -17.117606688793593,
        "E_en": -37.4415347898036,
        "E_kin": 14.864298452415749,
        "E_next": 0.0,
        "E_nn": 2.03010184805887,
        "E_nuc": 2.03010184805887,
        "E_tot": -15.087504840734724,
        "E_x": 0.0,
        "E_xc": -3.849942029201508,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 9.309571677795763,
        "E_eext": 0.0,
        "E_el": -17.117606688793593,
        "E_en": -37.4415347898036,
        "E_kin": 14.864298452415749,
        "E_next": 0.0,
        "E_nn": 2.03010184805887,
        "E_nuc": 2.03010184805887,
        "E_tot": -15.087504840734724,
        "E_x": 0.0,
        "E_xc": -3.849942029201508,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}