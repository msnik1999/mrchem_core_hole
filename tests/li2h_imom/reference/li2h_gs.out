***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         mom_merged                               ***
***         Git commit hash    aec5c05c74a8084661ef-dirty               ***
***         Git commit author  Niklas GÃ¶llmann                         ***
***         Git commit date    Mon Jun 3 10:21:30 2024 +0200            ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8

---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : mom
 Git commit hash       : 9ebc1da6290a65a252e6
 Git commit author     : Laura Ratcliff
 Git commit date       : Wed Jun 14 14:11:39 2023 +0200

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (8 threads)

---------------------------------------------------------------------------



===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================



***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :       -5.467375        2.657087        0.384125
    1      Li            :        2.486974        1.970663       -0.291434
    2       H            :       -0.093749       -1.621650       -3.769928
---------------------------------------------------------------------------
 Center of mass          :       -1.395671        2.047469       -0.211988
===========================================================================



***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                           3.000
 Beta electrons          :                                           3.000
 Total electrons         :                                           6.000
---------------------------------------------------------------------------
    n     Occ Spin       :                                            Norm
---------------------------------------------------------------------------
    0    2.00    p       :                              9.999990112179e-01
    1    2.00    p       :                              9.999938653129e-01
    2    2.00    p       :                              9.999998284835e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 14.864298452416
 E-N energy              :            (au)                -37.441534789804
 Coulomb energy          :            (au)                  9.309571677796
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -3.849942029202
 N-N energy              :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -17.117606688794
 Nuclear energy          :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.508750484073e+01
                         :      (kcal/mol)             -9.467552227533e+03
                         :        (kJ/mol)             -3.961223852000e+04
                         :            (eV)             -4.105519217094e+02
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                 -2.195382298641
    1    2.00    p       :            (au)                 -2.124392128750
    2    2.00    p       :            (au)                 -0.121805989151
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -8.883160833086
===========================================================================



***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :       -5.467375        2.657087        0.384125
    1      Li            :        2.486974        1.970663       -0.291434
    2       H            :       -0.093749       -1.621650       -3.769928
---------------------------------------------------------------------------
 Center of mass          :       -1.395671        2.047469       -0.211988
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 14.864298452416
 E-N energy              :            (au)                -37.441534789804
 Coulomb energy          :            (au)                  9.309571677796
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -3.849942029202
 N-N energy              :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -17.117606688794
 Nuclear energy          :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.508750484073e+01
                         :      (kcal/mol)             -9.467552227533e+03
                         :        (kJ/mol)             -3.961223852000e+04
                         :            (eV)             -4.105519217094e+02
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                 -2.195382298641
    1    2.00    p       :            (au)                 -2.124392128750
    2    2.00    p       :            (au)                 -0.121805989151
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -8.883160833086
===========================================================================


===========================================================================
                             Orbital Positions
---------------------------------------------------------------------------
Index                    :               x               y               z
---------------------------------------------------------------------------
    0                    :       -5.465635        2.656916        0.383952
    1                    :        2.485325        1.970808       -0.291291
    2                    :        0.009184       -1.359758       -3.516142
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        5.942250       -6.535930        6.846963
 Magnitude               :            (au)                       11.176297
                         :         (Debye)                       28.407314
---------------------------------------------------------------------------
 Nuclear vector          :       -9.034951       12.261601       -3.491855
 Magnitude               :            (au)                       15.625948
                         :         (Debye)                       39.717198
---------------------------------------------------------------------------
 Total vector            :       -3.092700        5.725672        3.355108
 Magnitude               :            (au)                        7.321534
                         :         (Debye)                       18.609484
===========================================================================




***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 15s                       ***
***                                                                     ***
***************************************************************************